lib/internal/tty.js

```diff
@@ -22,6 +22,12 @@
 
 'use strict';
 
+const {
+  StringPrototypeToLowerCase,
+  StringPrototypeSplit,
+  RegExpPrototypeTest
```

Comment:
Please use the ASCII order, and leave a trailing comma:
```suggestion
  RegExpPrototypeTest,
  StringPrototypeSplit,
  StringPrototypeToLowerCase,
```